AIPscan/Reporter/templates/report_ingest_log_tabular.html

```diff
@@ -51,7 +54,10 @@
                 <b>{{ row["User"] }}</b>
               </td>
               <td>
-                {{ row["AIPName"] }} <code>({{ row["AIP"] }})</code>
+                {{ row["AIPName"] }}
+              </td>
+              <td>
+                <code><span class="text-muted">{{ row["AIPUUID"] }}</span></code>
```

Comment:
With the AIP UUID is in it's own column, I think just `<td>{{ row["AIPUUID"] }}</td>` might suffice here. In the other tables, we don't have the values in that column muted.

Reply:
This is just a stylistic decision on my part here so I appreciate it might look redundant? 

![image](https://user-images.githubusercontent.com/1880412/100950607-b2127100-34da-11eb-8fe3-c1d0dced7d16.png)

I kinda like it. It's one way of adding flare. But can change.

Reply:
Yeah, fair enough! There should be room for some design choices here :smile:

Reply:
In general too, I think it's nice to have fixed width strings like a UUID in a fixed-width font for in a big enough list their alignment starts to get messy quite quickly. There would clearly be other ways to do this, but `<code>` is quite convenient.

Reply:
I was puzzling a bit at the use of the `<code>` block before, but this makes sense!